src/game/client/gameclient.h

```diff
@@ -11,8 +11,75 @@
 #include "render.h"
 #include "ui.h"
 
+// components
+#include "components/binds.h"
+#include "components/broadcast.h"
+#include "components/camera.h"
+#include "components/chat.h"
+#include "components/console.h"
+#include "components/controls.h"
+#include "components/countryflags.h"
+#include "components/damageind.h"
+#include "components/debughud.h"
+#include "components/notifications.h"
+#include "components/effects.h"
+#include "components/emoticon.h"
+#include "components/infomessages.h"
+#include "components/flow.h"
+#include "components/hud.h"
+#include "components/items.h"
+#include "components/mapimages.h"
+#include "components/maplayers.h"
+#include "components/menus.h"
+#include "components/motd.h"
+#include "components/nameplates.h"
+#include "components/particles.h"
+#include "components/players.h"
+#include "components/scoreboard.h"
+#include "components/skins.h"
+#include "components/sounds.h"
+#include "components/spectator.h"
+#include "components/stats.h"
+#include "components/voting.h"
```

Comment:
I really want to avoid this (also on ddnet). These includes force all components to be recompiled when a single component header file changes.